test/OpenTelemetry.Extensions.Enrichment.Tests/OpenTelemetry.Extensions.Enrichment.Tests.csproj

```diff
@@ -3,7 +3,9 @@
   <PropertyGroup>
     <Description>Unit test project for OpenTelemetry .NET SDK telemetry enrichment.</Description>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <TargetFrameworks>$(NetFrameworkMinimumSupportedVersion)</TargetFrameworks>
+    <TargetFrameworks>net7.0;net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);net48;net472;net471;net47;net462</TargetFrameworks>
```

Comment:
Be default in pipeline we are executing only net7.0,net6.0 and net462 (on Windows).

I would like to avoid keeping tests which are not executed in pipeline. See similar discussion here: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1403#discussion_r1363156763

Reply:
Got it, will update to match pipeline. I was following similar code also mentioned in that discussion.